acceptance-tests/dsl/src/main/java/org/hyperledger/besu/tests/acceptance/dsl/node/ProcessBesuNodeRunner.java

```diff
@@ -105,7 +106,21 @@ public void startNode(final BesuNode node) {
       params.add("--privacy-precompiled-address");
       params.add(String.valueOf(node.getPrivacyParameters().getPrivacyAddress()));
       params.add("--privacy-marker-transaction-signing-key-file");
-      params.add(node.homeDirectory().resolve("key").toString());
+      node.getPrivacyParameters()
```

Comment:
is this needed for this PR? I.e. does this PR make use of random vs signed Private Marker transactions?

Reply:
The PR requires the signing account to have balance.

Reply:
Not sure why this works in other places, but not now - maybe to do with Min-gas-price?
Will investigate other tests.

Reply:
Updated BesuNodeFactory with an option to supply a mining configuration, which then permits setting the gasPrice to zero and mitigates the need for this change.